presto-product-tests/src/main/java/io/prestosql/tests/hive/TestSqlStandardAccessControlChecks.java

```diff
@@ -221,6 +221,15 @@ public void testAccessControlSetTableAuthorization()
         bobExecutor.executeQuery(format("ALTER TABLE %s SET AUTHORIZATION alice", tableName));
     }
 
+    @Test(groups = {AUTHORIZATION, PROFILE_SPECIFIC_TESTS})
+    public void testAccessControlSetViewAuthorization()
+    {
+        assertThat(() -> bobExecutor.executeQuery(format("ALTER VIEW %s SET AUTHORIZATION bob", viewName)))
+                .failsWithMessage(format("Access Denied: Cannot set authorization for view default.%s to USER bob", viewName));
+        aliceExecutor.executeQuery(format("ALTER VIEW %s SET AUTHORIZATION bob", viewName));
+        bobExecutor.executeQuery(format("ALTER VIEW %s SET AUTHORIZATION alice", viewName));
+    }
```

Comment:
Please add a test with hive view (where views is defined in Hive).

Reply:
I get a `Hive views are not supported` when using hive views. I don't think they are supported in this setup.